client/post-editor/editor-seo-accordion/index.jsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Accordion from 'components/accordion';
+import AccordionSection from 'components/accordion/section';
+import CountedTextarea from 'components/forms/counted-textarea';
+import Gridicon from 'components/gridicon';
+import InfoPopover from 'components/info-popover';
+import TokenField from 'components/token-field';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteSlug } from 'state/sites/selectors';
+
+function EditorSeoAccordion( props ) {
+	const { translate, siteSlug } = props;
```

Comment:
No need to change here, but as future reference, you also have the option to inline the destructuring into the arguments of the function:

```js
function EditorSeoAccordion( { translate, siteSlug } ) {
```